src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -1311,3 +1311,34 @@ double GetStreamCpuLoad( PaStream * s )
 
     return PaUtil_GetCpuLoad( &stream->cpuLoadMeasurer );
 }
+
+/** Extensions */
+static void RenameStreamCb(pa_stream *s, int success, void *userdata)
+{
+    /* Currently does nothing but signal the caller. */
+    PaPulseAudio_Stream *l_ptrStream = (PaPulseAudio_Stream *) userdata;
+    pa_threaded_mainloop_signal( l_ptrStream->mainloop,
+                                 0 );
+}
+
+PaError PaPulseAudio_RenameStream( PaStream *s, const char *streamName )
+{
+    PaPulseAudio_Stream *stream = (PaPulseAudio_Stream *) s;
+    PaError result = paNoError;
+    
+    pa_threaded_mainloop_lock(stream->mainloop);
+    if (stream->inStream != NULL)
```

Comment:
spaces pa_threaded_mainloop_lock( stream->mainloop );

Reply:
OK, I think I got all of them. Let me know if there's any I missed.